infrastructure/oss/inline_value.go

```diff
@@ -0,0 +1,42 @@
+/*
+ * © 2023 Snyk Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package oss
+
+import (
+	"github.com/rs/zerolog/log"
+
+	"github.com/snyk/snyk-ls/domain/snyk"
+)
+
+func (oss *Scanner) GetInlineValues(path string, myRange snyk.Range) (result []snyk.InlineValue, err error) {
+	logger := log.With().Str("method", "GetInlineValues").Logger()
+	logger.Debug().Str("path", path).Msg("called")
+
+	inlineValues := oss.inlineValues[path]
+	if inlineValues == nil {
+		logger.Debug().Str("path", path).Msg("no inlineValues found")
+		return nil, nil
```

Comment:
Suggestion: Please return an empty slice here, otherwise [this](https://github.com/snyk/snyk-ls/pull/336/files#diff-a44b6d442dbb9afbfc2d35aa88806cfb9b0cb6c4a10508cb40e9c7404ec2fad6R83) will have an issue.